src/main/java/com/google/codeu/data/Marker.java

```diff
@@ -13,11 +14,16 @@
    * is placed.
    */
   public Marker(double lat, double lng, String content) {
```

Comment:
this String content here refers to the cinema's name and not the cinema's review right? Would you mind renaming it to something like cinemaName or something if thats the case?

Reply:
The content here refers to the marker content, which is essentially the cinema name since there is only cinema marker in our design. However, I am not sure if it is a good idea to specialise the naming to dedicate to cinema markers only.

Reply:
that's a good point, but I'm highlighting this because as someone who is reading this code for the first time, I am confused about whether it is referring to cinema name or cinema review. Why wouldn't it be a good idea?

Reply:
Because if we have other types of markers using this class, then having a cinema name would not make sense to those markers.